about.html

```diff
@@ -0,0 +1,134 @@
+          <div class="contact-us">
+            <p>Please contact us per Email for any further questions about HeavyFury festival 2024!<br>
+              <span>festival@HeavyFury.com</span>
+            </p>
+          </div>
+        </div>
+        <div class="logo-description">
+          <h3>HeavyFury rock & metal music festival 2024 logo</h3>
+          <p class="no-display">As every year, the winner of HeavyFury band battles will be able to add their bands attribute to next
+            years' logo!</p>
+          <p class="desktop-see">Introducing the captivating logo for the HeavyFury Music Festival, a thrilling celebration of all 
+            things rock and metal! Our logo showcases the essence of this electrifying event, featuring a powerful electric guitar 
+            at its core. With its bold and dynamic design, the logo embodies the raw energy and intensity that music enthusiasts crave.
+          </p>
+          <img src="images/heavyFury_logo.jpg">
+        </div>
+        <div class="see-past">
+          <h3>See past HeavyFury festivals</h3>
+          <p>Take a look at the last two HeavyFury music festivals that took place in Latvia and Estonia</p>
+          <div class="past-images"><div class="img-1">
+            <h3 class="year">2023</h3>
+            <p class="where">HeavyFury music festival Latvia</p>
+          </div><br>
+          <div class="img-2">
+            <h3 class="year">2022</h3>
+            <p class="where">HeavyFury music festival Estonia</p>
+          </div>
+          </div>
+        </div>
+        <div class="partners-div">
+          <h3>Partner</h3>
+          <div class="partners-img">
+            <img src="images/blackcraft logo.jpg" alt="logo">
+            <img src="images/Jackson_Guitars_Logo.png" alt="logo">
+            <img src="images/Metal-hammer-logo.png" alt="logo">
+            <img src="images/jagermeister-logo.png" alt="logo">
+            <img src="images/nuclear-blast-logo.png" alt="logo">
+          </div>
+        </div>
+        <footer>
+          <div class="about-footer">
+            <img src="images/heavyFury_logo.jpg">
+            <p class="no-display">2024 HeavyFury, Lithuania <br>
+              Some rights reserved</p>
+              <p class="about-footer-p">2024 HeavyFury, Lithuania
+                Some rights reserved <br><br>
+                Unless otherwise specified, all content of the HeavyFury festival website is protected 
+                under Lithuanian copyright law. Any unauthorized use, reproduction, or distribution of the 
+                content is strictly prohibited. For inquiries regarding the use of specific content, please 
+                contact us. We appreciate your respect for intellectual property and support 
+                of the HeavyFury festival.
+          </div>
+        </footer>
+      </div>
+    </div>
+    <div id="dynamicHeadliners" class="ignore">
+      <div id="see-more" class="ignore"></div>
+
+    </div>
```

Comment:
- [ ] As stated in the project requirement The pages should look almost identical to the original design, outside the content differences.But your About page is a little bit different from the design guidelines after the footer page for desktop screen size there is Headliners section as shown below so Kindly fix it. 
- Your About page footer page look likes this : 
![footer](https://github.com/agneta1991/Capstone_project/assets/93598189/270aa931-1a82-4373-83b9-ac8f80f626b1)

- The Design Guidelines About page Look likes this : 
![fotter 12](https://github.com/agneta1991/Capstone_project/assets/93598189/2770b810-a763-4c0f-a394-ee4c2821a9c6)